cmd/contrib/main.go

```diff
@@ -165,6 +168,17 @@ func newPlugin() error {
 
 	if result.ExampleCredential != "" {
 		result.ValueComposition = getValueComposition(result.ExampleCredential)
+		result.TestCredentialExample = result.ExampleCredential
+	} else {
+		// If no example credential has been provided, a random 30-character alphanumeric
+		// value is generated to be used within the test suite.
+		lettersAndDigits := []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890")
+		rand.Seed(time.Now().UnixNano())
+		randomString := make([]rune, 30)
+		for i := range randomString {
+			randomString[i] = lettersAndDigits[rand.Intn(len(lettersAndDigits))]
+		}
+		result.TestCredentialExample = string(randomString)
```

Comment:
If no value is provided as credential example, what do we want to use as a place-holder? I'm thinking any string that's randomised enough should be good enough (didn't use the `crypto` lib here - I don't see the need for cryptographic security for testing purposes).

Reply:
No need to care about it in this package, we already have a function in the `plugintest` package that handles this:
```go
plugintest.ExampleSecretFromComposition(schema.ValueComposition{
	Uppercase: true,
	Lowercase: true,
	Digits:    true,
})
```

Reply:
Run `make aws/example-secrets` to see this in action.